packages/frontend/src/contexts/game/reducer/index.js

```diff
@@ -58,6 +58,12 @@ export const gameReducer = (state, action) => {
         turnCount: state.turnCount + 1,
       };
 
+    case types.HANDLE_GAME_OVER:
+      return {
+        ...state,
+        stage: 'game over',
```

Comment:
Can we make the game stages into an object using `Object.freeze` so it's almost like an enum and can't be overridden?

Reply:
I'm imagining the game stages being formatted like the action types. 

```javascript
export const GAME_STAGES = Object.freeze({
  PLAYING: 'PLAYING',
  GAME_OVER: 'GAME_OVER',
  // ... other stages
})
```

Reply:
Thats cool. I'd never used Object.freeze before.